tests/test_sdds.py

```diff
@@ -89,18 +89,18 @@ def test_read_no_more(self):
 
     def test_read_header(self):
         test_head = b"""
-SDDS1
-!# big-endian
-&parameter name=acqStamp, type=double, &end
-&parameter name=nbOfCapTurns, type=long, &end
-&array name=horPositionsConcentratedAndSorted, type=float, &end
-&array
-    name=verBunchId,
-    type=long,
-    field_length=3,
-&end
-&data mode=binary, &end
-"""
+        SDDS1
+        !# big-endian
+        &parameter name=acqStamp, type=double, &end
+        &parameter name=nbOfCapTurns, type=long, &end
+        &array name=horPositionsConcentratedAndSorted, type=float, &end
+        &array
+            name=verBunchId,
+            type=long,
+            field_length=3,
+        &end
+        &data mode=binary, &end
+        """
```

Comment:
Does the indentation affect our reader? Or should it just be so in the mocked file?

Reply:
It does not. That's why I made it nicer to read in the tests.

Reply:
Alright, just wanted to confirm 👍